lib/bolt_server/transport_app.rb

```diff
@@ -440,8 +476,22 @@ def make_winrm_target(target_hash)
     # @param environment [String] the environment to fetch the list of tasks from
     get '/tasks' do
       in_pe_pal_env(params['environment']) do |pal|
-        tasks = pal.list_tasks
-        tasks_response = tasks.map { |task_name, _description| { 'name' => task_name } }.to_json
+        tasks_response = task_list(pal).to_json
+
+        # We structure this array of tasks to be an array of hashes so that it matches the structure
+        # returned by the puppetserver API that serves data like this. Structuring the output this way
+        # makes switching between puppetserver and bolt-server easier, which makes changes to switch
+        # to bolt-server smaller/simpler.
+        [200, tasks_response]
+      end
+    end
+
+    # Fetches the list of tasks for a bolt-project
+    #
+    # @param environment [String] the environment to fetch the list of tasks from
+    get '/bolt_project_tasks' do
```

Comment:
I guess we are going with `project` prepended, not `bolt_project`. :upside_down_face: